README.md

```diff
@@ -7,7 +7,7 @@ CNS Manager exposes APIs that can be invoked by authorized users to detect issue
 This repository provides artifacts for deploying CNS manager in vanilla Kubernetes cluster, as well as the client sdk to invoke its endpoints.
 
 ## Deploying cns-manager
-CNS manager needs to be deployed in one of the Kubernetes clusters in the vCenter.  
+CNS manager needs to be deployed in one of the namespaces running on [Supervisor Management Zones](https://techdocs.broadcom.com/us/en/vmware-cis/vsphere/vsphere-supervisor/8-0.html) in the vCenter.
```

Comment:
We don't plan to support "Storage vMotion for CNS volumes" on wcp, so all references in this readme file along with other references should be removed from this wcp branch.